<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqQueryClauseWidget</class>
 <widget class="QWidget" name="pqQueryClauseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
     <item>
      <widget class="QComboBox" name="criteria">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="condition">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is between</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is one of</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is &gt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is &lt;=</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="valueStackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <widget class="QWidget" name="singleValue">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="range">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value_min"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>and</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="value_max"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="location">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value_x"/>
         </item>
         <item>
          <widget class="QLineEdit" name="value_y"/>
         </item>
         <item>
          <widget class="QLineEdit" name="value_z"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="multiBlock">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value_block"/>
         </item>
         <item>
          <widget class="QToolButton" name="showCompositeTree">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="emptyValue">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_empty_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="locationWithTolerance">
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QLineEdit" name="location_y"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="location_x"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="location_z"/>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="location_tolerance">
           <property name="text">
            <string>0.1</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>but within epsilon</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>criteria</tabstop>
  <tabstop>condition</tabstop>
  <tabstop>location_x</tabstop>
  <tabstop>location_y</tabstop>
  <tabstop>location_z</tabstop>
  <tabstop>location_tolerance</tabstop>
  <tabstop>value_x</tabstop>
  <tabstop>value_y</tabstop>
  <tabstop>value_z</tabstop>
  <tabstop>value_min</tabstop>
  <tabstop>value_max</tabstop>
  <tabstop>value</tabstop>
  <tabstop>value_block</tabstop>
  <tabstop>showCompositeTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
